curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -580,7 +580,7 @@ class AsyncPrefetchCallback {
 
         int ret = s3Client_->GetDiskCacheManager()->WriteReadDirect(
             context->key, context->buf, context->len);
-        LOG_IF(ERROR, ret < 0)
+        LOG_IF(INFO, ret < 0)
```

Comment:
too many log also use LOG(INFO), maybe LOG_EVERY_SECOND will be better.

Reply:
fix